enterprise/server/test/integration/remote_execution/rbetest/rbetest.go

```diff
@@ -80,6 +79,9 @@ import (
 	scpb "github.com/buildbuddy-io/buildbuddy/proto/scheduler"
 	guuid "github.com/google/uuid"
 	bspb "google.golang.org/genproto/googleapis/bytestream"
+
+	// included to define flags
+	_ "github.com/buildbuddy-io/buildbuddy/enterprise/server/cmd/executor"
```

Comment:
Hm, wondering why this is this needed? Seems like if the flag is not transitively imported already, then setting the flag one way or the other would have no effect